_data/changelogs/component-combo-box.yml

```diff
@@ -2,6 +2,14 @@ title: Combo box
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Updated combo box result filtering.
+    summaryAdditional:  Now, search results are sorted by options that start with the query, followed by options that contain the query.
```

Comment:
**suggestion:** Use the word "sort" instead of "filter", since I don't think it has affected how it filters results. 

```suggestion
    summary: Updated how the component sorts results after typing.
    summaryAdditional:  Now, the component displays options that start with the query at the top of the list, followed by options that contain the query.
```